tests/test_response.py

```diff
@@ -81,3 +74,25 @@ def test_response_option_mimetype_init(monkeypatch):
     assert ro.static_media_types['.js'] == 'text/javascript'
     assert ro.static_media_types['.json'] == 'application/json'
     assert ro.static_media_types['.mjs'] == 'text/javascript'
+
+
+def test_response_set_stream(resp):
```

Comment:
Thanks for adding these tests, while I believe we already have tests for `Response.set_stream()`, the more the merrier. 

However, both of these cases look almost identical, would it be possible to DRY a bit using [`pytest.mark.parametrize(...)`](https://docs.pytest.org/en/7.1.x/how-to/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions)?

E.g.:
```python
@pytest.mark.parametrize('content', [b'', b'dummy content'])
def test_response_set_stream(resp, content): ...
```